src/canvascontext.ts

```diff
@@ -14,22 +14,24 @@ export class CanvasContext implements RenderContext {
   static get WIDTH(): number {
     return 600;
   }
+
   static get HEIGHT(): number {
     return 400;
   }
+
   static get CANVAS_BROWSER_SIZE_LIMIT(): number {
     return 32767; // Chrome/Firefox. Could be determined more precisely by npm module canvas-size.
   }
 
   static SanitizeCanvasDims(width: number, height: number): [number, number] {
-    if (Math.max(width, height) > this.CANVAS_BROWSER_SIZE_LIMIT) {
-      warn('Canvas dimensions exceed browser limit. Cropping to ' + this.CANVAS_BROWSER_SIZE_LIMIT);
-      if (width > this.CANVAS_BROWSER_SIZE_LIMIT) {
-        width = this.CANVAS_BROWSER_SIZE_LIMIT;
-        // note: Math.min return 0 for undefined, NaN for null. Would change inputs.
+    const limit = this.CANVAS_BROWSER_SIZE_LIMIT;
```

Comment:
This is just a small fix to improve readability (& reduce duplication).

Reply:
Thanks for this 👍